apps/web/components/dialog/EditLocationDialog.tsx

```diff
@@ -311,8 +312,15 @@ export const EditLocationDialog = (props: ISetLocationDialog) => {
                 query={locationsQuery}
                 success={({ data }) => {
                   if (!data.length) return null;
-                  const locationOptions = [...data].filter((option) => {
-                    return !isTeamEvent ? option.label !== "Conferencing" : true;
+                  const locationOptions = [...data].map((option) => {
+                    if (isTeamEvent) {
+                      // Let host's Default conferencing App option show for Team Event
+                      return option;
+                    }
+                    return {
+                      ...option,
+                      options: option.options.filter((o) => o.value !== OrganizerDefaultConferencingAppType),
```

Comment:
This is an alternate way to identify Default Conferencing App. Now All Video apps and Host's Default Conferencing app would come under Conferencing only.
User Events would just filter out 'Host's default App' option identified by `OrganizerDefaultConferencingAppType`

Reply:
cc @emrysal